subsys/logging/log_msg.c

```diff
@@ -270,6 +271,7 @@ void z_impl_z_log_msg_static_create(const void *source,
 
 	z_log_msg_finalize(msg, source, out_desc, data);
 }
+EXPORT_SYMBOL(z_impl_z_log_msg_static_create);
```

Comment:
Again, this is a global symbol - there is no implicit dependency on llext for logging.

Can we put this into `llext_export.c` as well?